docs/source/guides/network_node/ursula_configuration_guide.rst

```diff
@@ -17,6 +17,36 @@ The worker's ethereum account must remain unlocked to send automated work confir
 ether to pay for transaction gas; however, it is *not* necessary (and potentially risky) to hold NU tokens on a worker's
 account for any reason.
 
+
+Worker Requirements
+-------------------
+
```

Comment:
Here are the operation requirement factors:

- Level 2 Availability, NAT management, Firewalls, etc.
- Peer metadata storage (grows with network size, each node is `2.0K`) 
- 2GB of free RAM to derive scrypt passwords
- 1GB RAM for normal runtime (needs profiling)
- CPU for re-encryptions (extremely light weight, see CI benchmarks)

Reply:
Made some updates to the list:
- I think breaking down the RAM requirements is a bit overkill no? I can change 4GB (minimum) to 3GB (minimum) if you rather.
- re: HDD storage, I think we should still put a sane minimum - any thoughts there?

Reply:
Yeah - not meant as direct inclusions - just a way to think about the problem space and attempting to answer the question: *"What exactly dictates the requirements of nucypher?"*

Reply:
✔️